templates/helm-values-external-dns.tpl.yaml

```diff
@@ -0,0 +1,5 @@
+podAnnotations:
+  iam.amazonaws.com/role: ${external_dns_role_arn}
+extraArgs:
+  registry: txt
+  txt-owner-id: external-dns-controller
```

Comment:
Can you add the environment to this value so that staging won't try to delete production and vice-versa?